docs/rules/no-await-in-promise-methods.md

```diff
@@ -0,0 +1,42 @@
+# Disallow Promise methods containing an await
+
+🚫 This rule is _disabled_ in the ✅ `recommended` [config](https://github.com/sindresorhus/eslint-plugin-unicorn#preset-configs).
+
+🔧 This rule is automatically fixable by the [`--fix` CLI option](https://eslint.org/docs/latest/user-guide/command-line-interface#--fix).
+
+<!-- end auto-generated rule header -->
+<!-- Do not manually modify this header. Run: `npm run fix:eslint-docs` -->
+
+It's useless to use an await in a Promise all, allSettled, any, or race method and this reduces the code readability.
```

Comment:
It's not "useless", but it is most likely a mistake.

This description generally needs to be better written. Use `Promise.all()` syntax, etc.

Reply:
It's corrected.